docs/modules/servers/pages/distributed/run/run-docker.adoc

```diff
@@ -28,7 +28,7 @@ A default domain, james.local, has been created. You can see this by running:
 
 James will respond to IMAP port 143 and SMTP port 25.
 You have to create users before playing with james. You may also want to create other domains.
-Follow the 'Useful commands' section for more information about James CLI.
+Follow the xref:distributed/operate/cli.adoc['Useful commands'] section for more information about James CLI.
```

Comment:
CLI is likely generic: duplicate the page and make it a partial?

Reply:
Covered here: https://github.com/apache/james-project/pull/2326

Where the cli doc has been put in a partial in @vttranlina PR. 

Is it ok?

Reply:
Yes but we should not forget to update links when doing the rebase ;-)

Reply:
Hmm I mean there is a distributed and a postgres page for the cli, that both include a common partial.

Is that enough? Or by generify did you just want to completely isolate it?